runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsConfig.java

```diff
@@ -148,6 +148,61 @@ default Color getNonClanMemberColor()
 
 	@ConfigItem(
 		position = 10,
+		keyName = "drawClanCallerNames",
+		name = "Draw Clan caller names",
+		description = "Configures whether or not clan caller names should be drawn"
+	)
+	default boolean drawClanCallerNames()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		position = 11,
+		keyName = "clanCallerNames",
+		name = "Caller Names",
+		description = "Clan caller names separated by a comma"
+	)
+	default String getClanCallerNames()
+	{
+		return "";
+	}
+
+	@ConfigItem(
+		position = 12,
+		keyName = "clanCallerColor",
+		name = "Clan caller color",
+		description = "Color of clan caller names"
+	)
+	default Color getClanCallerColor()
+	{
+		return Color.ORANGE;
+	}
+
+	@ConfigItem(
+		position = 13,
+		keyName = "drawClanCallerTargetNames",
+		name = "Draw Clan caller target names",
+		description = "Configures whether or not clan caller Target names should be drawn"
+	)
+	default boolean drawClanCallerTargetNames()
```

Comment:
This could be changed to highlighted players target (based on my previous comment)